napari/_qt/_qapp_model/qactions/_help.py

```diff
@@ -0,0 +1,38 @@
+"""This module defines help menu actions.
+"""
+from typing import List
+
+from app_model.types import Action
+
+from ...._app_model.constants import CommandId, MenuId
+from ...dialogs.qt_about import QtAbout
+from ...qt_main_window import Window
+
+try:
+    from napari_error_reporter import ask_opt_in
+except ModuleNotFoundError:
+    ask_opt_in = None
+
+
+def _show_about(window: Window):
```

Comment:
@tlambert03 am I injecting the Window correctly here? the provider for Window is registered in the init, but I am getting a 
```
After injecting dependencies for NO arguments, _show_about() missing 1 required positional argument: 'window'
```

is there additional declaration I need to do to annotate this function or?

Reply:
With that correction you do look to be injecting it correctly now, so I'm not immediately sure why it's not working.  Need to pull and have a look to see why it would be any different 🤔